### PR TITLE
UpNext Implementation 

### PR DESCRIPTION
**Pull request by sveeravelli**

### CODE REVIEW

File: sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/BridgeMessageBuilder.java [hunk 1]
```diff
@@ -18,8 +18,8 @@
  * Created by michael.len on 2/1/16.
  */
 class BridgeMessageBuilder {
-
-  public static WritableMap buildTimeChangedEvent (OoyalaPlayer player){
+  public static String nextVideoEmbedCode;
```
Comment (lental) [inline sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/BridgeMessageBuilder.java]: why do you need to store this here? no storing anything in the BridgeMessageBuilder - it is entirely a static method class. no state in it

File: sdk/react/upNext.js [hunk 2]
```diff
@@ -101,15 +101,16 @@ var UpNext = React.createClass({
       var upNextWidth = ResponsiveDesignManager.makeResponsiveMultiplier(this.props.width, thumbnailWidth, [0.8,1], [520]);
       var countdown = this.renderCountdownTimer();
       var upNextImage = (
-      <TouchableHighlight style={[styles.thumbnailContainer, {width: upNextWidth}]}
```
Comment (lental) [inline sdk/react/upNext.js:104]: not a fan of this.  the entire image should be clickable.  why did you change this?

**Comment by sveeravelli** [comment]
Comments has been resolved.

File: sdk/react/upNext.js (hunk 2 above)
Comment (lental) [inline sdk/react/upNext.js:104]: fixed with style change on touchable highlight below
